### DISCUSSION TITLE
no longer works as expected

### DISCUSSION THREAD

**Discussion by Mitdan** [furybee/chrome-tab-modifier#406] [Q&A] [answered]
i used this extention to differentiate between different tabs from the same url that i use for different things, but until today (or sometime in the past few days) it closes all instances and keeps only one, while also changing the tab text to this new one. so i cant have multiple tabs from the same url with a different name for each. 

is this an update?  am i missing some setting that will allow me to use it as described?

**Comment by sylouuu** [comment, marked as answer]
You can disable the "unique" action inside your rule.

Bests

**Reply by Mitdan** [reply]
disabling the  "unique"  stopped closing the other instances, but it still renames all similar tabs with the same newest name

**Reply by Mitdan** [reply]
ok it might have been an issue with my browser, after updating chrome it works as expected as it were.
